rpa-builder/6.2/modules/ROOT/pages/toolbox-aws-operations-queries-with-aws.adoc

```diff
@@ -0,0 +1,118 @@
+= Queries With AWS
+
+The Queries with AWS Action Step enables you to extract specific data from a document by specifying queries in natural language so that the response contains the data you need, without having to apply further filtering on the JSON Output to get the relevant values. 
+
+For example, if you are processing a credit card statement, you can specify queries such as: 
+
+* "What is the customer name?"
+* "What is the credit card company name?"
+* "What is the account number?"
+* "What is the payment due date?"
+
+== Inbound Variables
+
+* *Alias text* 
```

Comment:
Some of these bullets are Sentence case and others are Title Case. Are they literal values? Is there a way to get the devs to make them consistent?

Reply:
They show as in the UI.